modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/block/processing/BlockValidator.scala

```diff
@@ -112,5 +116,22 @@ object BlockValidator {
             else
               none
         }
+
+      private val addressesLockedAtOrdinal: Map[Address, SnapshotOrdinal] = Map(
+        Address("DAG6LvxLSdWoC9uJZPgXtcmkcWBaGYypF6smaPyH") -> SnapshotOrdinal(1447110L) // NOTE: BitForex
+      )
```

Comment:
Please move this part to the `application.conf` (the resource-based configuration PR is already merged). Also, I think this configuration should be per environment (I assume you've seen my PR as it was open for a very long time, but if no - see how the last kryo ordinal or the last global snapshot ordinal are specified in the config). So, for mainnet, we should lock this address at ordinal X, but for testnet we should do that (or maybe not and we don't care) on a different ordinal.

Reply:
@marcinwadon That code is on _develop_ ... this is on a hotfix branch. Maybe moving it to `application.conf` can be done in a later PR?